grpc-interface-support/src/cpp.py

```diff
@@ -17,20 +17,356 @@
 import shutil
 import subprocess
 import tempfile
+from io import TextIOWrapper
 from pathlib import Path
+from typing import Callable, Dict, List, Optional
 
-from generator import GrpcInterfaceGenerator
+import shared_utils
+from generator import GrpcServiceSdkGenerator, GrpcServiceSdkGeneratorFactory
 from proto import ProtoFileHandle
-from shared_utils import conan_helper
-from velocitas_lib import get_package_path
+from shared_utils import conan_helper, templates
+from velocitas_lib import get_package_path, get_workspace_dir
 
 CONAN_PROFILE_NAME = "host"
 
 
-class CppGrpcInterfaceGenerator(GrpcInterfaceGenerator):  # type: ignore
+def capture_textfile_area(
+    file: TextIOWrapper,
+    start_line: str,
+    end_line: str,
+    map_fn: Optional[Callable[[str], str]] = None,
+) -> List[str]:
+    area_content: List[str] = []
+    is_capturing = False
+    for line in file:
+        if line.strip() == start_line:
+            is_capturing = True
+        elif line.strip() == end_line:
+            is_capturing = False
+        elif is_capturing:
+            line = line.rstrip()
+
+            if map_fn:
+                line = map_fn(line)
+
+            area_content.append(line)
+    return area_content
```

Comment:
Will move it to shared utils for now which is anyway an incubator for moving to velocitas_lib

Reply:
the problem I have with that is that I fear it will never end in the velocitas_lib, due to missing time or whatever. History showed, if we don't do it immediately, we will never do it...